Fixed issued #326 - UI Issue: Lesson Content Overlaps Sidebar on Learn Cypress

hadiimran00 opened the pull request:
Fixed this issue:
UI Issue: Lesson Content Overlaps Sidebar on Learn Cypress #326 https://github.com/cypress-io/cypress-realworld-testing/issues/326

a90100 commented:
When would this pull request be merged?

MikeMcC399 commented:
@a90100 

> When would this pull request be merged? thx @MikeMcC399

I don't know the answer to this question.

I closed the PRs that I had open in this repo because I got no response from the Cypress.io team.

hadiimran00 commented:
how do i merge my changes?

MikeMcC399 commented:
@hadiimran00 

> how do i merge my changes?

To merge changes you need write access to the repository, as it says in the status message above. You don't have write access. The Cypress.io team are the ones with write access and they are not responding. As I wrote before, I had the same problem.

As far as I'm aware, they have been unable to allocate anybody to look after these training materials.

jennifer-shehane reviewed (changes requested):
I'm still seeing this issue at some widths with the preview URL: 
![Screenshot 2024-12-02 at 2 10 19 PM](https://github.com/user-attachments/assets/0997a5c0-a565-40ab-9ae7-5c380792ab5b)

hadiimran00 commented:
Fixed

> I'm still seeing this issue at some widths with the preview URL: ![Screenshot 2024-12-02 at 2 10 19 PM](https://private-user-images.githubusercontent.com/1271364/391680420-0997a5c0-a565-40ab-9ae7-5c380792ab5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0NjczNDgsIm5iZiI6MTczNDQ2NzA0OCwicGF0aCI6Ii8xMjcxMzY0LzM5MTY4MDQyMC0wOTk3YTVjMC1hNTY1LTQwYWItOWFlNy01YzM4MDc5MmFiNWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTdUMjAyNDA4WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODZjNjkxODM2NDk1MzFiYmI2OTlhMjRiNGZlNTllODg2NmY2ZjQ5ZmVlMjBkMDI5N2ZiYTViODMyNDhhMmY0ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.fK8gKKRvlwvUCt6SjcAPGYYiZGSzZlK4cAouiyFT2rM)

Fixed

jennifer-shehane reviewed (changes requested):
@hadiimran00 Running these changes locally, I still see areas where the content overlaps the sidebar. Are you sure you committed all of the changes?

![Screenshot 2025-01-07 at 4 05 35 PM](https://github.com/user-attachments/assets/852f86cc-bbba-4fa2-ac58-04c034398feb)

jennifer-shehane commented:
Closing as this does not fix the issue.
